### PR TITLE
fix!: align tokens with figma variables

### PR DESCRIPTION
**Pull request by alisonailea**
**Related Issue:** #8310

## Summary

There were some misalignments between figma variables and tokens as well as some changes to names to better align with the naming schema.

Core tokens are a logical scale
Semantic tokens have meaningful names representing their usage.

BREAKING CHANGE:

- Box-shadow is now shadow
- Core font family tokens use the font name
- Remove unnecessary core tokens line-height, font-size, letter-spacing, paragraph-spacing as these can be exclusive to semantic and reference core size tokens
- Core size tokens now use their pixel size in their name
- Border-radius is now corner-radius
- Border-width removed unnecessary tokens none, sm, md, lg
- Success, info, warning, and danger colors are now prefixed with the name "status" 
  -   --color-status-info-hover

### CODE REVIEW

File: packages/calcite-design-tokens/src/semantic/space.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      },
+      "xxl": {
+        "value": "{core.size.default.20}",
+        "type": "spacing"
+      },
+      "xxxl": {
+        "value": "{core.size.default.32}",
+        "type": "spacing"
+      },
+      "overlap": {
```
Comment (jcfranco) [inline packages/calcite-design-tokens/src/semantic/space.json:44]: For my own curiosity, what would be a typical use case for this token?

**Review by jcfranco — APPROVED** [review]
👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾
👾🕹️🕹️🕹️🕹️👾🕹️👾👾👾👾👾🕹️👾🕹️🕹️🕹️🕹️👾👾🕹️🕹️🕹️👾👾🕹️🕹️👾👾🕹️👾👾👾🕹️👾🕹️🕹️🕹️🕹️👾🕹️👾
👾🕹️👾👾🕹️👾🕹️👾👾👾👾👾🕹️👾🕹️👾👾👾👾🕹️👾👾👾👾🕹️👾👾🕹️👾🕹️🕹️👾🕹️🕹️👾🕹️👾👾👾👾🕹️👾
👾🕹️🕹️🕹️🕹️👾🕹️👾👾🕹️👾👾🕹️👾🕹️🕹️🕹️👾👾👾🕹️🕹️👾👾🕹️👾👾🕹️👾🕹️👾🕹️👾🕹️👾🕹️🕹️🕹️👾👾🕹️👾
👾🕹️👾👾🕹️👾🕹️👾🕹️👾🕹️👾🕹️👾🕹️👾👾👾👾👾👾👾🕹️👾🕹️👾👾🕹️👾🕹️👾👾👾🕹️👾🕹️👾👾👾👾👾👾
👾🕹️👾👾🕹️👾👾🕹️👾👾👾🕹️👾👾🕹️🕹️🕹️🕹️👾🕹️🕹️🕹️👾👾👾🕹️🕹️👾👾🕹️👾👾👾🕹️👾🕹️🕹️🕹️🕹️👾🕹️👾
👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾

**Review by benelan — APPROVED** [review]

File: packages/calcite-design-tokens/src/semantic/space.json (hunk 1 above)
Comment (alisonailea) [reply packages/calcite-design-tokens/src/semantic/space.json:44]: I don't know. I copied it from Figma variables. Maybe @SkyeSeitz can give some clarity.

Comment (SkyeSeitz) [reply packages/calcite-design-tokens/src/semantic/space.json]: Yup definitely not typical. Currently just used in Card for padding to the end of header. This is true in code and Figma.

**Comment by geospatialem** [comment]
Verified in [calcite-design-tokens](https://www.npmjs.com/package/@esri/calcite-design-tokens) in `2.0.0`.

**Pull request merged**